fecgov/fec-cms
# Remove code related to scraping and indexing for search.gov i14y drawer

## Summary

- Resolves #5905

- Now that we have switched to sitemap indexing for search.gov, remove un needed code related to scraping and indexing for i14y drawer

- The outdated `instructions.md` is removed and documentation will be created in the fec-cms Wiki in this issue:
https://github.com/fecgov/fec-cms/issues/5842

- Also remove test affiliates in search.gov dashboard and add instuctions to Wiki on how to create a test affiliate and branch.

- [ ] After this is deployed, do this issue https://github.com/fecgov/fec-cms/issues/5923

### Required reviewers
one developer

## Impacted areas of the application

	modified:   ../.gitignore
	modified:   fec/settings/base.py
	deleted:    search/management/commands/index_pages.py
	deleted:    search/management/commands/scrape_cms_pages.py
	deleted:    search/management/commands/scrape_transition_pages.py
	deleted:    search/management/commands/scrape_web_app_pages.py
	deleted:    search/management/data/transition_pages.json
	deleted:    search/management/data/web_app_pages.json
	deleted:    search/management/instructions.md
	deleted:    search/tests/test_indexing.py
	deleted:    search/utils/search_indexing.py

-  

## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)

## Related PRs

Related PRs against other branches:
https://github.com/fecgov/fec-cms/pull/5845

## How to test
The circle build was successful when pushing the PR branch
The only way to really test this except to run pytest or  push it to dev space and see if any tests fail
